src/NServiceBus.Transport.SQS.AcceptanceTests/Publishing/When_publishing_wrapped_event.cs

```diff
@@ -22,7 +22,7 @@ public async Task Should_receive_event(byte[] payload)
                 .WithEndpoint<Publisher>(b => b.When(c => c.EndpointsStarted, session => session.Publish(new MyEventWithPayload() { Payload = payload })))
                 .WithEndpoint<Subscriber>(b => { })
                 .Done(c => c.GotTheEvent)
-                .Run(TimeSpan.FromSeconds(30));
+                .Run(TimeSpan.FromSeconds(45));
```

Comment:
What was the reason for making the timeout longer?

Reply:
It kept getting a timeout error when running the tests on the build server.